revocation/internal/ocsp/ocsp_test.go

```diff
@@ -196,6 +196,7 @@ func TestCheckStatusFromServer(t *testing.T) {
 	t.Run("ocsp request roundtrip failed", func(t *testing.T) {
 		client := testhelper.MockClient([]testhelper.RSACertTuple{revokableCertTuple, revokableIssuerTuple}, []ocsp.ResponseStatus{ocsp.Good}, nil, true)
 		server := "http://localhost.test"
+		//lint:ignore SA1012 nil context is used for testing
```

Comment:
Does the following work?
```suggestion
		//lint: ignore SA1012 nil context is used for testing
```

Reply:
https://staticcheck.dev/docs/configuration/#line-based-linter-directives